pkg/api/types.go

```diff
@@ -58,9 +58,18 @@ const (
 	//
 	// * added AgentRequest.LastPermit
 	//
-	// Currently the latest version.
+	// Last used in release version <todo>.
```

Comment:
> `<todo>`

Leaving this for now, will set it before merging

Reply:
And update `pkg/api/VERSIONING.md` probably

Reply:
Ah, yep! good catch - thanks!